From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Finkel <me@martinfinkel.com>
Date: Wed, 5 Aug 2026 17:00:00 +0300
Subject: [PATCH] contrib: use SourceForge redirector for mingw-w64

The netcologne mirror no longer resolves. Use SourceForge's stable project
endpoint so downloads are redirected to an available mirror.
---
 contrib/src/pthreads/rules.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/src/pthreads/rules.mak b/contrib/src/pthreads/rules.mak
index 2fc23f8a01..c935a76264 100644
--- a/contrib/src/pthreads/rules.mak
+++ b/contrib/src/pthreads/rules.mak
@@ -1,7 +1,7 @@
 # winpthreads, dxvahd
 
 MINGW64_VERSION := 10.0.0
-MINGW64_URL := $(SF)/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v$(MINGW64_VERSION).tar.bz2
+MINGW64_URL := https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v$(MINGW64_VERSION).tar.bz2
 MINGW64_HASH=2c35e8ff0d33916bd490e8932cba2049cd1af3d0
 MINGW64_GITURL := https://git.code.sf.net/p/mingw-w64/mingw-w64
 
-- 
2.50.1.windows.1

